learing_journal/scripts/initializedb.py

```diff
@@ -43,5 +46,11 @@ def main(argv=sys.argv):
     with transaction.manager:
         dbsession = get_tm_session(session_factory, transaction.manager)
         for entry in ENTRIES:
-            row = MyModel(title=entry['title'], body=entry['body'], date=entry['date'], id=entry['id'])
+            # intdented like this so the linter stopped complaining
+            row = Journal(title=entry['title'], body=entry['body'],
+                          date=datetime.datetime.strptime(entry['date'],
```

Comment:
Since you use this same datetime twice, you could just assign it to a variable and use that same variable for both `date` and `date_last_updated`.